packages/teams-js/src/public/meeting.ts

```diff
@@ -414,4 +418,14 @@ export namespace meeting {
     ensureInitialized(FrameContexts.sidePanel, FrameContexts.meetingStage);
     registerHandler('meeting.speakingStateChanged', handler);
   }
+
+  export function registerRaiseHandStateChangedHandler(
+    handler: (raiseHandState: IRaiseHandState | SdkError | null) => void,
```

Comment:
How about return type to be 2 parameters? `(error: SdkError | null, raiseHandState: IRaiseHandState | null)`. All other APIs we built follow the same format.

Reply:
I had made it that way based on the feedback received in the devDesign review. However, I synced with Trevor and he agreed that it should be separated out.